STYLEGUIDE.md

```diff
@@ -0,0 +1,28 @@
+As with code, technical writing benefits from consistent rules. Please follow this guide when adding or editing CircleCI documentation.
+
+# Structure
+
+- Don't use h1 headers. The title you choose in a doc's frontmatter will automatically be converted into an h1.
+- Add a table of contents to articles with more than 10 h2s.
+
+# Formatting
+
+- Single-spaced, never double-spaced.
+
+# Lists
+
+- Capitalize each list item.
+- Place periods at the end of each list item.
+
+# Bold
+
+- Use for terms that are defined.
+- Use for interactive elements, like buttons or links.
```

Comment:
For talking about buttons that are in the app or somewhere in a UI. I'll make this clearer.